### PR TITLE
(#43) add: create new cookie by `Cookies.Add(*http.Cookie)`

### PR DESCRIPTION
**Pull request by dannypsnl**

### CODE REVIEW

**Comment by dannypsnl** [comment]
The reason that cookie doesn't act like header is because header in request & response is different. But cookie is more like a global state for in the user's context. So write back is not design error.
But the code you wrote is pretty clean, that's true, I'm not pretty sure, but what about delete? If we want to delete a cookie, how the code like?

**Comment by shana0440** [comment]
I thinking maybe like this `Cookie.New("name").Forget()`

**Comment by dannypsnl** [comment]
Blocking by #75

**Comment by dannypsnl** [comment]
@shana0440 其實receive是對的R

File: response/response.go [hunk 1]
```diff
@@ -20,3 +28,16 @@ func (res *Response) WithHeaders(headers Headers) *Response {
 	}
 	return res
 }
+
+func (res *Response) Cookies(cs ...*cookie.Cookie) *Response {
+	for _, c := range cs {
+		res.cookies = append(res.cookies, c.Generate())
```
[the remark this answers is not in the document]
Comment (dannypsnl) [reply response/response.go]: Then they will get `*http.Cookie`, that won't be a valid argument of `Cookies`

[the remark this answers is not in the document]
Comment (shana0440) [reply response/response.go]: oh, right.

**Pull request merged**